packages/flutter/lib/src/widgets/form.dart

```diff
@@ -125,6 +136,16 @@ class Form extends StatefulWidget {
   /// will rebuild.
   final VoidCallback onChanged;
 
+  /// If [AutoValidateMode.onUserInteraction] this form will only auto-validate
```

Comment:
The first sentence of the doc string should talk about what this property is and for. the detail on how to use this property should come after.